src/store/Reducer.ts

```diff
@@ -13,16 +13,16 @@ export const reducer: Reducer<State, Actions.Action> = (
   action
 ): State => {
   switch (action.type) {
-    case Actions.InnerType.ErrorFetch:
+    case Actions.OuterType.SetError:
       return {
         ...state,
         loading: false,
-        hub: { count: 0 }, // TODO: Remove these fake values
+        error: action.message,
       };
-    case Actions.OuterType.Increment:
+    case Actions.OuterType.ClearError:
       return {
         ...state,
-        hub: { count: (state.hub?.count ?? 0) + 1 },
+        error: undefined,
```

Comment:
this just clears out the error message, so whatever error component we use, it'll have to watch for changes to error at the top level of the state "tree"